src/python_testing/TC_RR_1_1.py

```diff
@@ -36,6 +36,10 @@
 #
 
 
+def generate_controller_name(fabric_index: int, controller_index: int):
+    return f"RD{fabric_index}{str(chr(ord('A') + controller_index))}"
```

Comment:
nit:

```suggestion
    return f"RD{fabric_index}{string.ascii_uppercase[controller_index]))}"
```

assuming you `import string`